optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/mux.py

```diff
@@ -314,6 +314,10 @@ def iteritems(self):
         for key in iterkeys(self):
             yield key, self[key]
 
+    def items(self):
+        """ Slower implementation with the use of __getitem__ """
```

Comment:
Even though the docstring is correct, it's a duplicate of the `iteritems()` method.  IMO it'd make more sense if it mentioned that it's based on `iteritems()` instead of `__getitem__()` (indirectly).